Doc/library/typing.rst

```diff
@@ -1005,5 +1005,10 @@ The module defines the following classes, functions and decorators:
       if TYPE_CHECKING:
           import expensive_mod
 
-      def fun():
-          local_var: expensive_mod.some_type = other_fun()
+      def fun(arg: 'expensive_mod.SomeType') -> None:
+          local_var: expensive_mod.AnotherType = other_fun()
+
+   Note that the first type annotation must be enclosed in quotes, making it a
+   "forward reference", to hide the expensive_mod reference from the
```

Comment:
I would propose to use back-quotes for expensive_mod here. Like this:
```
``expensive_mod``
```
Otherwise LGTM.